### PR TITLE
Doc: add v1.3.0 changelog

### PR DESCRIPTION
**Pull request by SeanHai**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: CHANGELOG-1.3.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- [QoS of SnapshotCloneServer.](https://github.com/opencurve/curve/pull/303)
+- [Recover file in recycleBin.](https://github.com/opencurve/curve/pull/259)
+- [Clean chunks recycled to chunkfile pool background.](https://github.com/opencurve/curve/pull/322)
+- [Silent data CRC verification in the background.](https://github.com/opencurve/curve/pull/377)
+- [Periodic automatic cleaning of recyclebin.](https://github.com/opencurve/curve/pull/310)
+
+
+## optimization
+
+- Optimize ansible script, optimize build script, optimize log printing.
```
Comment (ilixiaocui) [inline CHANGELOG-1.3.md:18]: optimize optimize build script, optimize log printing.

File: CHANGELOG-1.3.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Previous change logs can be found at [CHANGELOG-1.2](https://github.com/opencurve/curve/blob/master/CHANGELOG-1.2.md)
+
+## new features
+
+- [Support discard to garbage collection.](https://github.com/opencurve/curve/pull/189)
+- [QoS of client.](https://github.com/opencurve/curve/pull/268)
+- [QoS of SnapshotCloneServer.](https://github.com/opencurve/curve/pull/303)
+- [Recover file in recycleBin.](https://github.com/opencurve/curve/pull/259)
+- [Clean chunks recycled to chunkfile pool background.](https://github.com/opencurve/curve/pull/322)
```
Comment (ilixiaocui) [inline CHANGELOG-1.3.md:11]: fill chunks  in chunkfile pool with 0 in the background.

File: CHANGELOG-1.2.md [hunk 3]
```diff
@@ -22,6 +22,35 @@ Previous change logs can be found at [CHANGELOG-1.1](https://github.com/opencurv
     - [Support to list the volume on copyset where majority copy is failed.](https://github.com/opencurve/curve/pull/233)
 - [Clean up the unit test temporary folder.](https://github.com/opencurve/curve/pull/206/)
 
+
+Hardware: 6 nodes, each with:
```
Comment (cw123) [inline CHANGELOG-1.2.md:26]: 6 nodes as chunkserver

Don't need hardware configuration of client or mds ？

File: CHANGELOG-1.3.md (hunk 1 above)
Comment (SeanHai) [reply CHANGELOG-1.3.md:18]: > optimize optimize build script, optimize log printing.

done

File: CHANGELOG-1.3.md (hunk 2 above)
Comment (SeanHai) [reply CHANGELOG-1.3.md:11]: > fill chunks in chunkfile pool with 0 in the background.

done

File: CHANGELOG-1.2.md (hunk 3 above)
Comment (SeanHai) [reply CHANGELOG-1.2.md:26]: > 6 nodes as chunkserver
> 
> Don't need hardware configuration of client or mds ？

Yes, the environmental information is from test report and consistent with the previous version

**Review by cw123 — APPROVED** [review]

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**